### PR TITLE
Emails with Numbers in the domain are incorrectly rejected

### LINKED ISSUE #135: Emails with Numbers in the domain are incorrectly rejected

### PR DESCRIPTION
**Pull request by fdeeken**
fix #135

### CODE REVIEW

File: app/src/main/ui/src/i18n/de.json [hunk 1]
```diff
@@ -75,6 +75,7 @@
   "newGroup.notification.success": "Die Gruppe {{groupName}} wurde erfolgreich angelegt.",
   "newUser.notification.error": "Der Nutzer '{{username}}' konnte nicht gespeichert werden.",
   "newUser.notification.errorDuplicateMail": "Ein Nutzer mit der E-Mail '{{mail}}' existiert bereits.",
+  "newUser.notification.errorInvalidMail": "Bitte eine Gültige E-mail Address angeben.",
```
Comment (nfranzeck) [inline app/src/main/ui/src/i18n/de.json:78]:
```suggestion
  "newUser.notification.errorInvalidMail": "Bitte eine gültige E-Mail Adresse angeben.",
```

File: app/src/main/ui/src/i18n/en.json [hunk 2]
```diff
@@ -75,6 +75,7 @@
   "newGroup.notification.success": "The group {{groupName}} was created successfully.",
   "newUser.notification.error": "Could not save user '{{username}}'.",
   "newUser.notification.errorDuplicateMail": "A user with the E-Mail '{{mail}}' already exists.",
+  "newUser.notification.errorInvalidMail": "Please enter a valid E-mail.",
```
Comment (nfranzeck) [inline app/src/main/ui/src/i18n/en.json:78]:
```suggestion
  "newUser.notification.errorInvalidMail": "Please enter a valid e-mail address.",
```

**Review by nfranzeck — CHANGES_REQUESTED** [review]
Only typos in translations - good job

File: app/src/main/ui/src/i18n/en.json (hunk 2 above)
Comment (fdeeken) [reply app/src/main/ui/src/i18n/en.json:78]: LGTM

**Pull request merged**